### DISCUSSION TITLE
AI DENTE PRO Salinig Mode does not work properly

### DISCUSSION THREAD

**Discussion by mirkostella99** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1007] [Q&A] [answered]
When I use connect the Mac to Mag Safe at a lowest % of the limit (my limit is set to 80%) sailing mode shut (setted to 40%) down and the Mac continue charging until it reaches 80/84%. How's that possible? I upgraded to Pro version in order to have the Sailing Mode function but like this is just like have a free version... I got a Macbook M1 Pro 2021, everything is updated and Apple Battery Optimization is off.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @mirkostella99,

Unfortunately, our current version AlDente 1.22 has a Sailing Mode bug, which causes the Sailing Mode not to survive a restart. However, we are in the final stages of testing a new update that will fix this bug again.

Best,
Matthias

**Comment by mirkostella99** [comment]
Super, just upgraded and it works perfect!
